prow/jobs/busola/busola-tests/busola-tests.yaml

```diff
@@ -0,0 +1,109 @@
+# Code generated by rendertemplates. DO NOT EDIT.
+
+
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
+job_template: &job_template
+  decorate: true
+  path_alias: github.com/kyma-project/busola
+  max_concurrency: 10
+  labels:
+    preset-dind-enabled: "true"
+    preset-sa-gcr-push: "true"
+    preset-docker-push-repository-kyma: "true"
+  run_if_changed: "^tests/|^components/|^shared/|^package.json"
```

Comment:
do we want to run tests on the busola repo? I think those tests are useless at this point